.gitlab-ci.yml

```diff
@@ -0,0 +1,21 @@
+include:
+  - project: 'city-of-helsinki/kuva/ci-cd-config/ci-configuration'
+    ref: v2
+    file: '/.gitlab-ci-template.yml'
+
+variables:
+  APP_MIGRATE_COMMAND: /app/.prod/on_deploy.sh
+
+build:
+  extends: .build
+
+review:
+  extends: .review
+  variables:
+    K8S_SECRET_DEBUG: true
+    K8S_SECRET_DJANGO_SECRET_KEY: "$GL_QA_DJANGO_SECRET_KEY"
```

Comment:
Setting this for `review` job shouldn't be necessary.

Reply:
I've made it intentionally so that if `DJANGO_SECRET_KEY` is not set, the application will crash, as seen here: https://github.com/City-of-Helsinki/events-helsinki-cms/blob/master/project/settings.py#L15 .. so unless we'd give it a default value which I would be worried about its security, this can't be removed if I've understood everything correctly 😊